Updated welcome message with new committee and code of conduct

JamesDearlove opened the pull request:
Fixes #384
Fixes #269

mitchmcdee commented on uqcsbot/scripts/welcome.py:
```diff
@@ -10,9 +10,16 @@
     "This is the first time I've seen you, so you're probably new here",
     "I'm UQCSbot, your friendly (open source) robot helper",
-    "We've got a bunch of generic channels (e.g. #banter, #games, #projects) along with many " \
+    "We've got a bunch of generic channels (e.g. #banter, #memes, #projects) along with many " \
```
would prefer to keep games and choose between either banter or memes for diversity sake 👍

JamesDearlove commented:
Fixed up that channel and fixed a small typo 👍 
![welcome updated](https://user-images.githubusercontent.com/39483549/46989079-fdbd0180-d13e-11e8-91fa-def9bc48cb1e.png)

mitchmcdee commented on uqcsbot/scripts/welcome.py:
```diff
@@ -19,7 +19,7 @@
     "Your friendly committee and admins are Nicholas Lambourne @ndl, Jack Caperon @jam, Bradley Stone @Σ, " \
     "Joshua Sutton @jsutton, Ella de Lore @Ella, Matthew Lake @mgtlake, Ishraque Zahin @Ishraque, " \
```
wrap their slack names in brackets

mitchmcdee commented on uqcsbot/scripts/welcome.py:
```diff
@@ -10,7 +10,7 @@
+    "We've got a bunch of generic channels (e.g. #banter, #games, #projects) along with many " \
     "subject-specific ones (e.g #csse1001, #math1051, #csse2310)",
     "To find and join a channel, tap on the channels header in the sidebar",
     "The UQCS slack is a community for anyone who likes code, computers and pizza so " \
```
capitalise to `Slack`

mitchmcdee commented on uqcsbot/scripts/welcome.py:
```diff
@@ -19,7 +19,7 @@
     "Joshua Sutton @jsutton, Ella de Lore @Ella, Matthew Lake @mgtlake, Ishraque Zahin @Ishraque, " \
     "Ryan Kurz @lighthou and Raghav Mishra @rm",
-    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up"
+    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up",
```
remove the `to see what is coming up` (redundant given the start of the sentence imo)

JamesDearlove commented:
Changes have been added
![welcome 2](https://user-images.githubusercontent.com/39483549/46992935-f6eaba80-d14f-11e8-9613-5f47a800ecb4.png)
